src/scss/grommet-core/_objects.content-card.scss

```diff
@@ -0,0 +1,183 @@
+$card-hover-color: #EBEBEB;
+$social-twitter-color: #1DA1F2;
+$social-facebook-color: #3B5998;
+$social-linkedin-color: #0077B5;
+
+@mixin social-icon-color($social-color) {
+  fill: $social-color;
+  stroke: $social-color;
+}
+
+@mixin truncate-paragraph($truncate-p-line-height, $truncate-p-font-size) {
+  max-height: ($truncate-p-line-height * $truncate-p-font-size * 8);
+  // p line-height * (# lines)
+
+  &::after {
+    margin-top: ($truncate-p-line-height * $truncate-p-font-size * 7);
+    // p line-height * (# lines - 1)
+  }
+
+  @include media-query(palm) {
+    max-height: ($truncate-p-line-height * $truncate-p-font-size * 5);
+
+    &::after {
+      margin-top: ($truncate-p-line-height * $truncate-p-font-size * 4);
+    }
+  }
+}
+
+.content-card {
+  @include media-query(palm) {
+    padding: 0;
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $border-color;
+    }
+  }
+
+  .#{$grommet-namespace}control-icon-watch,
```

Comment:
I think this icon styling should be in _objects.icon.scss, as the behavior seems orthogonal to the content-cart context.